torch/optim/_muon.py

```diff
@@ -52,21 +52,21 @@ def _zeropower_via_newtonschulz(
         raise ValueError("Input tensor gradient must be a 2D matrix")
     if len(ns_coefficients) != 3:
         raise ValueError("Coefficients must be a tuple of exactly 3 values")
-    a, b, c = ns_coefficients[0], ns_coefficients[1], ns_coefficients[2]
-    X = grad.bfloat16()
+    a, b, c = ns_coefficients
+    ortho_grad = grad.bfloat16()
     if grad.size(0) > grad.size(1):
-        X = X.T
+        ortho_grad = ortho_grad.T
     # Ensure spectral norm is at most 1
-    X = X / (X.norm() + eps)
+    ortho_grad.div_(ortho_grad.norm().clamp(min=eps))
     # Perform the NS iterations
     for _ in range(ns_steps):
-        A = X @ X.T
-        B = b * A + c * A @ A
-        X = a * X + B @ X
+        gram_matrix = ortho_grad @ ortho_grad.T
```

Comment:
noob q just for curiosity: what does gram stand for here?

Reply:
missed this comment yesterday.. thanks Jiaming!